bodypix_tflite/README.md

```diff
@@ -0,0 +1,12 @@
+# BodyPix Tensorflow-lite Inference
+First install tensorflow-lite.
+
+```shell
+pip install --extra-index-url https://google-coral.github.io/py-repo/ tflite_runtime
```

Comment:
Does this mean you are actually using the Coral TPU?

Reply:
Hi

I check in TensorFlow website to install tflite package in python and found this command. This Tensorflow-Lite can use without Coral TPU.